xlml/apis/task.py

```diff
@@ -69,14 +69,30 @@ def run(self) -> DAGNode:
     with TaskGroup(
         group_id=self.task_test_config.benchmark_id, prefix_group_id=True
     ) as group:
-      provision, queued_resource, ssh_keys = self.provision()
+      provision, queued_resource, ssh_keys, _ = self.provision()
       run_model = self.run_model(queued_resource, ssh_keys)
       post_process = self.post_process()
       clean_up = self.clean_up(queued_resource)
       provision >> run_model >> post_process >> clean_up
 
     return group
 
+  def run_with_gcs_name_generation(self) -> DAGNode:
+    with TaskGroup(
+        group_id=self.task_test_config.benchmark_id, prefix_group_id=True
+    ) as group:
+      
+      provision, queued_resource, ssh_keys, gcs_location = self.provision()
+      run_model = self.run_model(
+          queued_resource, ssh_keys, {"OUTPUT_DIR": gcs_location}
```

Comment:
I see this `OUTPUT_DIR` used multiple places, can we have a enum variable somewhere to re-use to avoid typo?